src/App.js

```diff
@@ -1,39 +1,43 @@
 import React from 'react';
-import { BrowserRouter, Switch, Route,Link } from 'react-router-dom'
+import { BrowserRouter, Switch, Route, Link } from 'react-router-dom'
 import TodoContainer from './TodoContainer'
+import styled from './App.module.css'
+import { style } from '@mui/system';
 
 function App() {
 
   return (
     <BrowserRouter>
-     <nav>
-        <ul>
-      <li>
-      Home
-      </li>
-          <li>
-            <Link to="/list-1">List 1</Link>
-          </li>
-          <li>
-            <Link to="/list-2">List 2</Link>
-          </li>
-        </ul>
-      </nav>
-      <Switch>
+     
```

Comment:
indentation and white space in this return statement is all over the place, should be checked over again